scripts/publish_gatk_tool_wdls.sh

```diff
@@ -0,0 +1,113 @@
+#!/bin/bash
+#
+# A script to generate the GATK tool WDLs for a specified GATK release, and publish them
+# to the GATK tool WDL github repository broadinstitute/gatk-tool-wdls.
+#
+# Must be run from the root of a GATK clone.
+#
+# Usage: bash scripts/publish_gatk_tool_wdls.sh <gatk_version_tag>
+#
+# Eg.,   bash scripts/publish_gatk_tool_wdls.sh 4.1.9.0
+#
+# The specified GATK version will also be used to tag the WDLs themselves in
+# broadinstitute/gatk-tool-wdls
+#
+
+if [ $# -ne 1 ]; then
+  echo "Usage: $0 <gatk_version_tag>"
+  exit 1
+fi
+
+GATK_VERSION="$1"
+GATK_CLONE_ROOT=$( pwd )
+GATK_WDL_REPO="git@github.com:broadinstitute/gatk-tool-wdls.git"
+WDL_STAGING_AREA="wdl_staging_area"
+WDL_GEN_DIR="build/docs/wdlGen"
+DOCKSTORE_YML=".dockstore.yml"
+
+function cleanup() {
```

Comment:
To do this sort of thing I usually recommend trapping error/exit signals as well, but this is probably fine for now.